example/avoidance/avoidance.cpp

```diff
@@ -0,0 +1,175 @@
+/**
+ * @file avoidance.cpp
+ *
+ * @brief Example that demonstrates the usage of Avoidance plugin.
+ * The example registers with the FakeAvoidanceMediator which keeps track
+ * of the current trajectory. Callbacks with new trajectory information from PX4 are
+ * uploaded to the mediator. The mediator then "calculates" a new trajectory
+ * in real time and uploads that trajectory to the vehicle.
```

Comment:
Next to the example it would be good to also have some sort of integration test in order to prevent breaking this in the future.